### PR TITLE
Sponsorship Packet

### PR DESCRIPTION
**Pull request by InahL**
Purpose: Linking sponsorship packet enables website visitors to access and view sponsorship packets without leaving the website.

- Implemented PDF embedding feature for sponsorship packets.
- Integrated two new sponsorship packet designs.
- Ensured smooth display of PDFs on the web pages.

### CODE REVIEW

File: frontend/src/Pages/SponsorUsPage/SponsorUsPage.jsx [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+          display: 'flex',
+          flexDirection: 'column',
+          alignItems: 'center'
+          }}
+          >
+          <h3 style={{ 
+            textAlign: 'center'
+            }}
+            >
+              Want to support us?
```
Comment (maryellenmarino) [inline frontend/src/Pages/SponsorUsPage/SponsorUsPage.jsx:37]: Can you change the font here to be Poppins? It comes from the `fonts.css` so you can specify it like that.

**Comment by maryellenmarino** [comment]
This looks awesome! After you get the font updated I'll resolve the issues from my code I merged into main (it breaks everyone's pages for the most part) and then I can get this merged in!

File: frontend/src/Pages/SponsorUsPage/SponsorUsPage.jsx (hunk 1 above)
Comment (InahL) [reply frontend/src/Pages/SponsorUsPage/SponsorUsPage.jsx:39]: I changed it!

**Comment by maryellenmarino** [comment]
Just updated this branch to be consistent with the routing on the main branch.

**Review by alin600 — APPROVED** [review]
Looks good. Gtm.

**Pull request merged**